benchmarks/viirs_sdr_benchmarks.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Benchmark VIIRS SDR operations.."""
+
+from pyspectral.rayleigh import check_and_download as download_luts
+from pyspectral.rsr_reader import check_and_download as download_rsr
+
+
+class VIIRSSDR:
```

Comment:
Codebeat complains rightfully about the number of methods in this class. Could we make subclasses, eg one for single channels and one for composites?

Reply:
:thinking: I guess I just went with your structure on the AHI tests. I can clean this up.

Reply:
This should be done.